doc/rst/index.rst

```diff
@@ -98,15 +114,15 @@ the clock line is high (see :ref:`i2c_stop_bit`).
 |newpage|
 
 Connecting to the xCORE device
```

Comment:
Possibly this should be at the higher heading level with under/overlines of asterisks.

Reply:
This seems to match the original document, think it makes sense.

Reply:
ok, I'll leave it as is then